Pyfhel/PyPtxt.pyx

```diff
@@ -228,6 +228,7 @@ cdef class PyPtxt:
             content: (:obj:`bytes`) Python bytes object containing the PyPtxt.
             scheme: (:obj: `str`) String or type describing the scheme:
               * ('int', 'integer', int, 1, scheme_t.bfv) -> integer scheme.
+              * ('int', 'integer', int, 1, scheme_t.bgv) -> integer scheme.

```

Comment:
Change to ` (3, scheme_t.bgv) -> integer scheme.`. We will leave BFV as default integer scheme.